modules/ROOT/pages/config-alerts.adoc

```diff
@@ -6,92 +6,151 @@ endif::[]
+    # the alert annotations
+    annotations:
+        description: |
+        Cluster CPU usage exceeds 80%.
+
+See the https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[Alerting Rules] documentation for more details about Prometheus alerts.
+
+* To create an alert, run:
++
+----
+gravity resource create alert.yaml
+----
 
-If you apply changes to the `kapacitor-alerts` config map, you must restart Kapacitor. Follow the instructions in the Restart Kapacitor section.
+* To view existing alerts run:
++
+----
+gravity resource get alerts
+----
 
-== Configure Alerts
+* To remove an alert, run:
++
+----  
+gravity resource rm alert cpu-alert
+----
 
-. Configure the `FROM` and `TO` email addresses:
-.. Log in to the Ops Center console and then select *Configuration*.
-.. From the *Namespace* drop-down list, select *monitoring*.
-.. From the *Config maps* drop-down list, select *alerting-addresses*.
-.. Select *From* and then enter the `FROM` email address used for the alert.
-.. Select *To* and then enter the `TO` email address used for the alert.
+== Configure Alerts Delivery
 
-. Configure your SMTP email server to send alerts.
+To configure Alertmanager to send email alerts, create the following gravity resources. 
+
+. Using the following spec, create a file named `smtp-config.yaml` inside gravity, and replace the placeholder values with those of your SMTP configuration:
 +
-Configuring the SMTP server in Anypoint Access Management restarts the Anypoint Platform service responsible for initiating alerts.
-. Verify alerts configuration:
+[source,copy]
+----    
+kind: smtp
+version: v2
+metadata:
+    name: smtp
+spec:
+   host: <SMTP_HOST>
+   port: <SMTP_PORT>
+   username: <SMTP_USERNAME>
+   password: <SMTP_PASSWORD>
+---
+kind: alerttarget
+version: v2
+metadata:
+    name: email-alerts
+spec:
+    # email address of the alerts recipient
+    email: <RECIPIENT_EMAIL>
+----
+
+. Run `gravity resource create smtp-config.yaml`. You should see the following output:
 +
-** Verify that your SMTP server can send and receive emails using the addresses you configure as the `FROM` and `TO` addresses in the following section.
-** Verify that your cluster nodes can communicate with your SMTP server.
+----     
+Created cluster SMTP configuration
+Created monitoring alert target "email-alerts"
+----
+
+. Add a default router to the Alertmanager configuration inside gravity:
 +
-For example, use `telnet` to connect to your SMTP server from one of your cluster nodes:
+----
+kubectl get secret -n monitoring alertmanager-main -o json | jq --arg foo "$(kubectl get secret -n monitoring alertmanager-main -o json | jq -r '.data["alertmanager.yaml"]' | base64 -d | yq r - --tojson | jq -r '.route.routes[1] |= . + {"match":{"alertname": "Watchdog", "receiver": "default", "continue": true}}' | jq -r '.route.routes[0].match += {"continue":true}' | yq r - -P | base64 | tr -d '\n')" '.data["alertmanager.yaml"]=$foo' | kubectl apply -f -
```

Comment:
The command output looks good. 

It would be nice to have a before/after contents of alertmanager.yaml value when the above command is run. This will help the end users to notice what section was updated or added.

Reply:
I think that would provide customer with low-level details about the alerting implementation. They shouldn't have to see that IMO